strings.txt

```diff
@@ -18124,3 +18153,119 @@
     it = Scarica una volta le mappe per l’area desiderata e usa la navigazione e le funzioni di ricerca senza una connessione internet.
     fr = Téléchargez une seule fois les cartes de la zone désirée, et utilisez les fonctions de navigation et de recherche sans connexion Internet.
     ja = 希望するエリアの地図を一回ダウンロードするだけで、インターネット接続なしでもナビゲーションや検索の機能を使用できます。
+
+  [onboarding_notifications_message]
+    tags = ios, android
+    en = To receive the latest maps, get updates notifications
+    ru = Получайте уведомления об обновлениях, чтобы всегда иметь актуальные карты
+    tr = En son haritaları almak için güncelleme bildirimlerini alın
+    sv = Få avisering om uppdateringar för de senaste kartorna
+    nb = Få oppdateringsvarsler for å motta de nyeste kartene
+    ko = 최신지도를 받으려면, 업데이트 알림 받기
+    id = Untuk menerima peta-peta terbaru, dapatkan pemberitahuan pembaruan
+    hu = Fogadj értesítő üzeneteket a legfrissebb térképekről
+    de = Um die neuesten Karten zu erhalten, aktivieren Sie bitte die Update-Benachrichtigungen
+    fi = Ota päivitysilmoitukset käyttöön saadaksesi tuoreimmat kartat
+    nl = Ontvang update-meldingen om de nieuwste kaarten te ontvangen
+    da = For at modtage de nyeste kort kan du få opdateringsnotifikationer
+    cs = Chcete-li získat nejnovější mapy, pak zapněte upozornění na aktualizace
+    zh-Hans = 为了接收最新的地图，请获取更新通知
+    zh-Hant = 為了接收最新的地圖，請獲取更新通知
+    ar = تابع إخطارات التحديث للحصول على أحدث الخرائط
+    th = เพื่อรับแผนที่ล่าสุดและรับการแจ้งเตือนอัปเดต
+    uk = Отримуйте повідомлення про оновлення, щоб завжди мати актуальні мапи
+    pt = Para receber os mapas mais recentes, receba notificações de atualizações
+    es = Para recibir los mapas más recientes, obtén las notificaciones de actualización
+    ro = Pentru a obține cele mai recente hărți, obțineți notificările de actualizări
+    pl = Aby pobierać najnowsze mapy, włącz powiadomienia o aktualizacjach
+    sk = Ak chcete získavať najnovšie mapy, zapnite si notifikácie aktualizácií
+    it = Per ricevere le mappe aggiornate, attiva le notifiche di aggiornamento
+    fr = Pour recevoir les dernières cartes, recevez des notifications de mise à jour
+    ja = 最新地図の入手のためにアップデート通知を受信    
+
+  [onboarding_notifications_title]
+    tags = ios, android
+    en = Enable updates notifications
+    ru = Разрешите отправку уведомлений
```

Comment:
Ничего, что это не соответствует действительности?

> On 17.3.2016, at 12.37, Igor Tomko <notifications@github.com> wrote:
> 
> In strings.txt:
> 
> > +    ar = تابع إخطارات التحديث للحصول على أحدث الخرائط
> > +    th = เพื่อรับแผนที่ล่าสุดและรับการแจ้งเตือนอัปเดต
> > +    uk = Отримуйте повідомлення про оновлення, щоб завжди мати актуальні мапи
> > +    pt = Para receber os mapas mais recentes, receba notificações de atualizações
> > +    es = Para recibir los mapas más recientes, obtén las notificaciones de actualización
> > +    ro = Pentru a obține cele mai recente hărți, obțineți notificările de actualizări
> > +    pl = Aby pobierać najnowsze mapy, włącz powiadomienia o aktualizacjach
> > +    sk = Ak chcete získavať najnovšie mapy, zapnite si notifikácie aktualizácií
> > +    it = Per ricevere le mappe aggiornate, attiva le notifiche di aggiornamento
> > +    fr = Pour recevoir les dernières cartes, recevez des notifications de mise à jour
> > +    ja = 最新地図の入手のためにアップデート通知を受信    
> > +
> > +  [onboarding_notifications_title]
> > +    tags = ios, android
> > +    en = Enable updates notifications
> > +    ru = Разрешите отправку уведомлений
> Алекс, там идет просьба разрешить слать push-уведомления, но под предлогом общих уведомлений об устаревших данных.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>